snippets/Modal.md

```diff
@@ -123,14 +118,26 @@ function Modal(props){
 ```jsx
 //Add the component to render function
 
-ReactDOM.render(
-  <Modal 
-    isVisible={false}
-    title= "Modal Title"
-    content={<p>The content goes here</p>}
-    footer={<button>Cancel</button>}        
-    onClose ={()=> } //Add action to hide modal
-/>,
-  document.getElementById('root')
-);
+function App() {
+  const [ isModal, setModal] = React.useState(false);
+  
+  return (
+    <React.Fragment>
+      <button onClick={()=> setModal(true)}>xx</button>
+      <Modal 
+        isVisible={ isModal }
+        title= "Modal Title"
+        content = {<p>Add your content here</p>}
+        footer = {<button>Cancel</button>}
+        onClose ={()=> setModal(false)}
+      />
+    </React.Fragment>
+  )
+}
+
+ReactDOM.render( <App/>, document.getElementById('root'));
 ```
+
+<!-- tags: modal -->
```

Comment:
Please use existing tags, mainly. Using the component's name as a tag is not really all that helpful.